lib/reality.rb

```diff
@@ -28,7 +28,8 @@ def self.reload!
   end
 
   # extras
-  require_ %w[extras/open_weather_map extras/geonames]
+  require_ %w[extras/open_weather_map extras/geonames extras/economics]
```

Comment:
Hey. Didn't you note I've changed extra modules naming from "extras/{topic}" to "extras/{service}". You should!